### ISSUE TITLE
v1.4.0: isn't adhering to styleExt default config setting

### ISSUE THREAD

**Issue by denver-HJS** [angular/angular-cli#7662]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
```
@angular/cli: 1.4.0
node: 7.2.1
os: win32 x64
@angular/animations: 4.3.6
@angular/cdk: 2.0.0-beta.10
@angular/common: 4.3.6
@angular/compiler: 4.3.6
@angular/core: 4.3.6
@angular/flex-layout: 2.0.0-beta.9-f978b94
@angular/forms: 4.3.6
@angular/http: 4.3.6
@angular/material: 2.0.0-beta.10
@angular/platform-browser: 4.3.6
@angular/platform-browser-dynamic: 4.3.6
@angular/router: 4.3.6
@angular/cli: 1.4.0
@angular/compiler-cli: 4.3.6
@angular/language-service: 4.3.6
typescript: 2.5.2
```
### Repro steps.
Run the command to generate a new component (see the attached screen shot for more detail).

`ng g component components/commodity-input`

*.angular-cli.json*:
```
"defaults": {
    "styleExt": "scss",
    "component": {},
    "serve": {
      "port": 9000
    }
  }
```

### Desired functionality.
I would like the newly generated component to have a SCSS file created (this was previously working)

![angular-cli-styles-bug](https://user-images.githubusercontent.com/1916018/30291337-68d0682e-96e7-11e7-8b37-97ec6b201526.png)

**Comment by mjolk** [comment]
I don't know about the import stuff but can confirm ng g component creates style files with css extension instead of scss despite styleExt setting.

**Comment by denver-HJS** [comment]
Looks like the styleExt setting is a dupe of #7644  #7647. Leaving this one open for visibility to the 'named module' issue

**Comment by Brocco** [comment]
Correct the styleExt issue has been resolved, leaving open due to the named module issue.

@denver-HJS in the future, please open separate issues... thanks!

**Comment by denver-HJS** [comment]
Yep, sure thing. I realized I should've separated them into separate issues after the OP. Thanks for looking into it!
